guides/source/action_text_overview.md

```diff
@@ -8,149 +8,141 @@ rich text content.
 
 After reading this guide, you will know:
 
-* How to configure Action Text.
-* How to handle rich text content.
-* How to style rich text content and attachments.
+* What action text is and how to install and configure Action Text.
```

Comment:
Instead of mentioning action text in one sentence twice, maybe: "What Action Text is, and how to install and configure it"